.travis.yml

```diff
@@ -1,19 +1,19 @@
 language: python
 sudo: false
 env:
-  - DJANGO_VERSION=1.10.4
-  - DJANGO_VERSION=1.9.12
-  - DJANGO_VERSION=1.8.17
+  - DJANGO_VERSION=1.11.*
+  - DJANGO_VERSION=1.10.*
+  - DJANGO_VERSION=1.9.*
+  - DJANGO_VERSION=1.8.*
 python:
   - "2.7"
   - "3.4"
   - "3.5"
   - "3.6"
-  - "nightly" # currently points to 3.7-dev
+  - "nightly"
 install:
   - pip install -e .
-  - pip install -q Django==$DJANGO_VERSION
-  - pip install -rrequirements-dev.txt
+  - pip install -Ur requirements-dev.txt Django==$DJANGO_VERSION
```

Comment:
For the backstory: https://github.com/elbuo8/sendgrid-django/commit/af1ea06846d0ce6ce1aa9c53ed5003904fecd595

Travis has pytest installed by default, but it's pinned to an old version that we need to update. I believe this may have also applied to a few other lines in `requirements-dev.txt`, where Travis had them pre-installed but they wouldn't match what you'd normally get in a fresh environment.